### ISSUE TITLE
1.18.2 Can't add non-IE fluid outputs to Resource Packs 

### ISSUE THREAD

**Issue by JainaProudgurl** [BluSunrize/ImmersiveEngineering#5336]
### Description of the issue:
Was making a modpack using a plethora of 1.18.2 mods and wanted to add cracking recipes to turn Diesel > Kero > Gas > LPG to the IE Refinery. 

For some reason, whenever I add outputs that are not IE, it doesn't seem to work. It always just defaults to an output of air when I look at the recipe in JEI. 

Here's the formatting that I used in the file. 
https://pastebin.com/XsEavwk0

Note, its only the output that's the issue. Regardless of what I put in the results ("tag","fluid","fluidTag"), it always fails to recognize the pnuematiccraft fuel in the output. What's weird is that the same isn't true for the inputs; regardless of which "input" type I used ("tag","fluid","fluidTag") it always worked and showed it in JEI. I'm not really sure if the bug is possible related to JEI or IE. I know its not likely a bug related to Pnuematicraft, because I tested with fluids from almost every other mods and I ran into the same thing as above.

### Versions & Modlist
IE Version: 1.18.2-8.0.2-149
Forge Version: 40.1.51
Pnuematiccraft Version: 1.18.2-3.2.4-93
JEI Version: 1.18.2-9.7.0.196

Couldn't get the modlist to generate using the F3+C thing mentioned. I listed the mods that I thought were relevant to this issue, but if there's another way to get a full modlist I'd be more than happy to grab it.

**Comment by malte0811** [comment]
Fluid outputs do not use tags, you need to specify the fluid directly there. So
```json
"result":{"fluid":"pneumaticcraft:gasoline","amount":12}
```
should work. Only the tag is in the Forge namespace, the fluid itself is under PnC.

**Comment by JainaProudgurl** [comment]
Like I said, it doesn't matter what I used; I tried fluid and it didn't work either (and yes, I changed it to what you had above)

I ended up copying a bad example, which was completely my fault. The one that I started with looked like exactly what you just posted.

Here's a different file that I used that also didn't work, and it uses the format you just used. 
https://pastebin.com/j2jaVKQP

As I stated, the recipe shows up, but the output just shows as air. Everything else displays correctly.
